openstf/monitoring/teams.py

```diff
@@ -97,15 +97,38 @@ def post_teams_alert(msg, invalid_coefs=None, coefsdf=None, url=None):
     config = ConfigManager.get_instance()
     # Add invalid coefficients and manual coefficients-query to message
     if invalid_coefs is not None and coefsdf is not None:
-        invalid_coefs_str = "".join(
+        # add invalid coefficient information to message in dict-format
+        invalid_coefs_text = "".join(
             [
-                f"\ncoefficient name: {row.coef_name}, new value: {row.coef_value_new}, last value: {row.coef_value_last}"
+                f"\n* **{row.coef_name}**: {round(row.coef_value_new, 2)}, (previous: {round(row.coef_value_last, 2)})"
                 for index, row in invalid_coefs.iterrows()
             ]
         )
         query = build_sql_query_string(coefsdf, "energy_split_coefs")
-        msg = f"{msg}. {invalid_coefs_str}\nIf you would like to update the coefficients manually, use this query:\n {query}"
-
+        query_text = "If you would like to update the coefficients manually in the database, use this query:"
```

Comment:
Line length? It probably a good idea to set a right margin in your editor so its easy to see when lines are too long: 
![image](https://user-images.githubusercontent.com/6715707/109676629-2fe92580-7b79-11eb-9db5-bdf0bfcf8989.png)

Reply:
I have an automatic black formatter, which take care of things, but it does not automatically split up long strings and docstrings. I have added a ruler for manual corrections.